src/components/tableOfTasks/TaskTable.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Typography, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Box,
+} from '@mui/material';
+import { TaskRow } from '../index.ts';
+import theme from '../../theme/index.ts';
```

Comment:
the `import/extensions` rule is disabled in aslant, so u don't need to type extension.
```suggestion
import { TaskRow } from '../index';
import theme from '../../theme';
```